app/models/bestuurseenheid.js

```diff
@@ -26,6 +26,12 @@ export default class Bestuurseenheid extends Model {
     inverse: null,
   })
   bestuursorganen;
+  @belongsTo('recognized-worship-type', {
+    async: true,
+    inverse: null,
+    optional: true,
```

Comment:
I don't think `optional` is needed? It's the first time I see it and I can't find it in the API docs :smile: . Did a warning / error message tell you to add it or something?

Reply:
Also, if the relationship isn't actually used I wouldn't even add it to the model class. Less code to maintain in the future.